### DISCUSSION TITLE
Artifact naming questions: Version without `v`

### DISCUSSION THREAD

**Discussion by stdedos** [goreleaser/goreleaser#5015] [Q&A] [answered]
Can I has `.Version`, but without the `v` stripped?

If you check above, I was "medicating" that via `.Tag`.
However, I discovered that `snapshot.name_template` "overrides" the `.Version`.

So, ... this way (using `.Tag`) I'll need to fill my config with `{{ if .IsSnapshot }}SNAPSHOT-{{ .Summary }}_{{ else }}v{{ .Tag }}{{ end }}` (and "give up" the `snapshot.name_template`)

**Comment by caarlos0** [comment, marked as answer]
easier way is probably `{{ trimprefix .Version "v" }}`

https://goreleaser.com/customization/templates/#functions

**Reply by caarlos0** [reply]
or change `snapshot.name_template`
